selfdrive/car/toyota/interface.py

```diff
@@ -321,7 +331,7 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):  # py
 
     # removing the DSU disables AEB and it's considered a community maintained feature
     # intercepting the DSU is a community feature since it requires unofficial hardware
-    ret.communityFeature = ret.enableGasInterceptor or ret.enableDsu or smartDsu
+    ret.communityFeature = ret.enableGasInterceptor or ret.enableDsu or smartDsu or candidate is CAR.ALPHARD_TSS2
```

Comment:
Why community feature?

Reply:
Didn't I read that being a "supported" car has strict requirements, like Comma needs to have one it its fleet and have done the tuning?

Reply:
from README:
> To promote a car from community maintained, it must meet a few requirements. We must own one from the brand, we must sell the harness for it, has full ISO26262 in both panda and openpilot, there must be a path forward for longitudinal control, it must have AEB still enabled, and it must support fingerprinting 2.0

from a not-too-recent article (https://comma-ai.medium.com/how-to-write-a-car-port-for-openpilot-7ce0785eda84):
> STEP 5 — Get the openpilot pull request merged
> At comma’s discretion, some car ports can be upstreamed into the openpilot repo and become officially supported, while other might maintain the status of “Community Maintained Cars” indefinitely. It mainly depends on the code quality/complexity of the pull request, and on the popularity of the car/the presence of such a brand in comma’s fleet.